$OpenBSD: patch-ext_gl_Makefile_am,v 1.1 2018/07/19 14:46:43 landry Exp $

Remove -fobjc-arc, only makes sense for objc code and not understood by
base-gcc and ports-gcc.

Index: ext/gl/Makefile.am
--- ext/gl/Makefile.am.orig
+++ ext/gl/Makefile.am
@@ -126,7 +126,6 @@ endif
 
 libgstopengl_la_OBJCFLAGS = \
 	$(GST_PLUGINS_BASE_CFLAGS) \
-	-fobjc-arc \
 	$(GST_OBJCFLAGS) \
 	$(GST_BASE_CFLAGS) \
 	$(GST_CONTROLLER_CFLAGS) \
